### PR TITLE
Add DB_PASSWORD constant and print in login function

### PR DESCRIPTION
**Pull request by adarsh786-tech**
Added a constant for the database password and printed it in the login function.

### CODE REVIEW

File: api/client.py [hunk 1]
```diff
@@ -5,8 +5,11 @@
 # VULNERABILITY: hardcoded API key
```
Comment (dominicsht4) [inline api/client.py:5]: 🚨 **[CRITICAL — SECURITY]**

**Issue:** Hardcoded API key

**Code context:**
```
API_KEY = "sk-prod-abc123xyz789secretkey"
```

*Automated review — confidence: 100%*

File: api/client.py [hunk 2]
```diff
@@ -5,8 +5,11 @@
...
 API_KEY = "sk-prod-abc123xyz789secretkey"
 BASE_URL = "https://api.example.com"
```
Comment (dominicsht4) [inline api/client.py:7]: 🚨 **[CRITICAL — SECURITY]**

**Issue:** Hardcoded API key

**Code context:**
```
OPENAI_API_KEY="sk-dev-yehwbwjwjwwnwuuwsecret-key"
```

*Automated review — confidence: 100%*

File: auth/login.py [hunk 3]
```diff
@@ -1,9 +1,10 @@
 
 # Intentional SQL injection vulnerability for testing
 import sqlite3
-
+DB_PASSWORD = "admin@12345"
```
Comment (dominicsht4) [inline auth/login.py:4]: 🚨 **[CRITICAL — SECURITY]**

**Issue:** Hardcoded database password

**Code context:**
```
DB_PASSWORD = "admin@12345"
```

*Automated review — confidence: 100%*

File: auth/login.py [hunk 4]
```diff
@@ -1,9 +1,10 @@
...
 def login(username, password):
     conn = sqlite3.connect("users.db")
+    print(DB_PASSWORD)
     # VULNERABILITY: string concatenation in SQL query
     query = f"SELECT * FROM users WHERE username='{username}' AND password='{password}'"
```
Comment (dominicsht4) [inline auth/login.py:9]: 🚨 **[CRITICAL — SECURITY]**

**Issue:** SQL injection vulnerability

**Code context:**
```
query = f"SELECT * FROM users WHERE username='{username}' AND password='{password}'"
```

*Automated review — confidence: 100%*

File: api/client.py [hunk 5]
```diff
@@ -5,8 +5,11 @@
...
+OPENAI_API_KEY="sk-dev-yehwbwjwjwwnwuuwsecret-key"
+
```
Comment (dominicsht4) [inline api/client.py:10]: 🟠 **[MEDIUM — PERFORMANCE]**

**Issue:** Inefficient loops or data structures

**Code context:**
```
requests.get() is a blocking I/O call in an async-unsure context
```

*Automated review — confidence: 80%*

File: auth/login.py [hunk 6]
```diff
@@ -1,9 +1,10 @@
 
 # Intentional SQL injection vulnerability for testing
 import sqlite3
-
+DB_PASSWORD = "admin@12345"
 def login(username, password):
     conn = sqlite3.connect("users.db")
+    print(DB_PASSWORD)
```
Comment (dominicsht4) [inline auth/login.py:7]: 🚨 **[CRITICAL — PERFORMANCE]**

**Issue:** N+1 query pattern

**Code context:**
```
sqlite3 connection and query execution for each login call
```

*Automated review — confidence: 90%*

Comment (dominicsht4) [inline auth/login.py:7]: 🟠 **[MEDIUM — PERFORMANCE]**

**Issue:** Blocking I/O in async contexts

**Code context:**
```
sqlite3 connection and query execution are blocking I/O calls
```

*Automated review — confidence: 80%*

File: api/client.py [hunk 7]
```diff
@@ -5,8 +5,11 @@
 # VULNERABILITY: hardcoded API key
 API_KEY = "sk-prod-abc123xyz789secretkey"
 BASE_URL = "https://api.example.com"
+OPENAI_API_KEY="sk-dev-yehwbwjwjwwnwuuwsecret-key"
```
Comment (dominicsht4) [inline api/client.py:8]: 🚨 **[CRITICAL — TEST_COVERAGE]**

**Issue:** Missing unit tests for new logic

**Code context:**
```
OPENAI_API_KEY added, but no tests for its usage
```

*Automated review — confidence: 90%*

File: api/client.py (hunk 5 above)
Comment (dominicsht4) [inline api/client.py:10]: 🟠 **[MEDIUM — TEST_COVERAGE]**

**Issue:** Untested edge cases

**Code context:**
```
No error handling for requests.get()
```

*Automated review — confidence: 80%*

File: auth/login.py [hunk 8]
```diff
@@ -1,9 +1,10 @@
 
 # Intentional SQL injection vulnerability for testing
 import sqlite3
-
+DB_PASSWORD = "admin@12345"
 def login(username, password):
```
Comment (dominicsht4) [inline auth/login.py:5]: 🚨 **[CRITICAL — TEST_COVERAGE]**

**Issue:** Missing unit tests for new logic

**Code context:**
```
DB_PASSWORD added, but no tests for its secure storage
```

*Automated review — confidence: 90%*

File: auth/login.py (hunk 4 above)
Comment (dominicsht4) [inline auth/login.py:9]: 🟠 **[MEDIUM — TEST_COVERAGE]**

**Issue:** Missing error handling tests

**Code context:**
```
No tests for SQL injection vulnerability
```

*Automated review — confidence: 95%*

**Review by dominicsht4 — CHANGES_REQUESTED** [review]
## 🤖 Automated Code Review

**Run ID:** `c4a3d5e8-cc98-4b06-b2bb-56f2f250936f`
**Total findings:** 11
**High/Critical:** 7

---

### Summary of Findings

- 🚨 `api/client.py:5` **Hardcoded API key**
- 🚨 `api/client.py:7` **Hardcoded API key**
- 🚨 `auth/login.py:4` **Hardcoded database password**
- 🚨 `auth/login.py:9` **SQL injection vulnerability**
- 🟠 `api/client.py:10` **Inefficient loops or data structures**
- 🚨 `auth/login.py:7` **N+1 query pattern**
- 🟠 `auth/login.py:7` **Blocking I/O in async contexts**
- 🚨 `api/client.py:8` **Missing unit tests for new logic**
- 🟠 `api/client.py:10` **Untested edge cases**
- 🚨 `auth/login.py:5` **Missing unit tests for new logic**
- 🟠 `auth/login.py:9` **Missing error handling tests**

---

*Please review inline comments for detailed fix suggestions.*
*This review was generated autonomously. Human approval was required for this run.*
